admin/broadleaf-open-admin-platform/src/main/java/org/broadleafcommerce/openadmin/web/service/FormBuilderServiceImpl.java

```diff
@@ -964,13 +958,7 @@ protected ListGrid createListGrid(String className, List<Field> headerFields, Li
                         recordField.setDisplayValue(p.getDisplayValue());
                     } else {
                         if (headerField.getFieldType().equals("DATE")) {
-                            try {
-                                Date date = new SimpleDateFormat("yyyy.MM.dd HH:mm:ss").parse(p.getValue());
-                                String newValue = formatter.format(date);
-                                recordField.setValue(newValue);
-                            } catch (Exception ex) {
-                                recordField.setValue(p.getValue());
-                            }
+                            this.setDateToRecordField(recordField, p);
```

Comment:
you create a new formatter in this method, I don't see why do we need it to create in loop, extract it and pass as a parameter if you need this method